notebooks/Chiwei Yan - Cutting Stock.html

```diff
@@ -1022,6 +1022,11 @@ <h4 id="Method-2:-Branch-and-Bound-on-Root-Node">Method 2: Branch-and-Bound on R
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>
     <!-- MathJax -->
+    <script type="text/x-mathjax-config">
+        MathJax.Hub.Config({
+          tex2jax: {inlineMath: [['$','$'], ['\\(','\\)']]}
+        });
+    </script>        
```

Comment:
Why do we need it?

Reply:
MathJax, by default does not use the `$...$` latex math delimiters. They need to be enabled by this script. I tried to convert the notebooks to HTML using nbconvert and it added the script to enable the `$...$` delimiter on its own. Since these files were created 3 years ago, probably not enabling the delimiters was a bug that has been fixed now.

Reply:
All these notebooks have to be updated anyway, and we will need to reconvert them. This will fix the issue. If we want to fix the problem until the time the notebooks are updated, we can use this approach of adding the enabling script. Otherwise, these HTML pages can be left as is till the notebooks are updated and reconverted. 

Please let me know what you decide.

Reply:
Nice that the bug was fixed and the notebooks updated!